### ISSUE TITLE
Failing to build on Rocky 8

### ISSUE THREAD

**Issue by williameveretteggplant** [gnustep/libs-base#395]
Starting with commit: 4515e4a28b63ccd3e715cb21736e95242d671843
libs-base is failing to build on Rocky 8 with this message:
`checking FFI library usage... configure: error: The ffi library (libffi) does not appear to be working.  Perhaps it's missing or you need a more recent version.  Version 3.0.9 or later should work, and you can find a link to it n the list of packages for download at http://www.gnustep.org/resources/sources.html`
I can confirm that does build on Ubuntu 22. 
Now, Rocky 8 only has libffi 3.1, while Ubuntu 22 has libffi 3.5. But 3.1 should be sufficient. Also, this message could be red herring, hiding another reason.
Does anybody know what is going on?

**Comment by gcasa** [comment]
> Starting with commit: 4515e4a28b63ccd3e715cb21736e95242d671843
> 
> libs-base is failing to build on Rocky 8 with this message:
> 
> `checking FFI library usage... configure: error: The ffi library (libffi) does not appear to be working.  Perhaps it's missing or you need a more recent version.  Version 3.0.9 or later should work, and you can find a link to it n the list of packages for download at http://www.gnustep.org/resources/sources.html`
> 
> I can confirm that does build on Ubuntu 22. 
> 
> Now, Rocky 8 only has libffi 3.1, while Ubuntu 22 has libffi 3.5. But 3.1 should be sufficient. Also, this message could be red herring, hiding another reason.
> 
> Does anybody know what is going on?

This could be due to whatever path libffi is located at not being present in the LD_LIBRARY_PATH at build time.

**Comment by williameveretteggplant** [comment]
It looks like @qmfrederik is already looking into this...

**Comment by qmfrederik** [comment]
Yes, the culprit is https://github.com/gnustep/libs-base/commit/4515e4a28b63ccd3e715cb21736e95242d671843; has a https://github.com/gnustep/libs-base/pull/389 fix but I believe @rfm wanted to have a closer look at this.  @williameveretteggplant  if you cherry-pick e0af8f8035ca468db34ef9827c67294d3942b498 it should unblock you.
